### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Update docs to reflect changes made today

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -110,45 +117,50 @@ servo_one.position
 
 Note that this value is not in degrees!
 
-By default, servos will be un-powered when your robot starts, and can freely rotate. Once you set a value, they then fix to that rotation.
+By default, servos will be un-powered when your robot starts, and can freely rotate. Once you set a value, they then fix to that rotation. This also happens when you turn off your robot, or unplug your USB.
```
Comment (kierdavis) [inline index.md]: Could you make "unplug your USB" a bit more specific?

File: index.md [hunk 2]
```diff
@@ -110,45 +117,50 @@ servo_one.position
...
 
-## Cameras (Vision Boards)
-As always, a dictionary of the available cameras is obtained, then an individual camera is obtained using the serial number, or if there is only a single camera connected, then there is a function to get that as follows:
+### Ultrasound
+The ultrasound sensor can be used to measure the distance your robot is away from something.
 
+Assuming you've wired up the read (_echo_) and write (_trigger_) pins to 6 and 7:
```
Comment (kierdavis) [inline index.md]: I'd recommend listing these in the order they're passed to the function (trigger before echo) - less chance of confusion that way

Comment (kierdavis) [inline index.md]: Maybe also comment on what it returns? I.e. distance in metres

Comment (kierdavis) [inline index.md]: Ignore above comment, this is already in the test further on

File: reference.md [hunk 3]
```diff
@@ -2,39 +2,29 @@
...
-Class that represents a marker.
-- **Methods**
-  - `is_token_marker()`: if the marker is a token marker.
-  - `is_wall_marker()`: if the marker is a wall marker.
-
+### robot.PinValue
+Holds constant the values of a pin
+- **Values**
+  - `HIGH`: The pin is receiving a high current
+  - `LOW`: The pin is receiving a low current
```
Comment (kierdavis) [inline reference.md]: Please change "current" to "voltage"

Maybe also give examples of a "high" and "low" voltage? i.e. 5V and 0V respectively

File: reference.md [hunk 4]
```diff
@@ -91,6 +67,10 @@ Class that represents the robot as a whole
 ---
 ###  robot.Servo
 Class that represents a servo.
+- **Methods**
+  - `read_analogue()`: Get a list of all the values of the analogue pins, indexed by pin number.
+  - `read_ultrasound(trigger_pin, echo_pin)`: Get the distance reported on your ultrasound sensor, with the read and write pins connected to `echo_pin` and `trigger_pin` respectively. The value will be in metres.
```
Comment (kierdavis) [inline reference.md]: Again, I think it would be clearer to keep the order of the pins consistent (trigger before echo)

**Comment by RealOrangeOne** [comment]
Merging to get this live now, will fix comments later.

**Pull request merged**